### ISSUE TITLE
Option to name the output executable

### ISSUE THREAD

**Issue by oberluz** [ofek/pyapp#74]
Hi,

Would it be possible to add an  option to name rge executable based on the project name rather than pyapp.exe?
Congratulations by the way, really useful took.

**Comment by Brandontam29** [comment]
Yes, just set the `PROJECT_EXE_NAME` environment variable to the name of your choice. Here is an example.

```python
PROJECT_EXE_NAME: str = "custom_name"
```

**Comment by hasansezertasan** [comment]
> Yes, just set the `PROJECT_EXE_NAME` environment variable to the name of your choice. Here is an example.
> 
> PROJECT_EXE_NAME: str = "custom_name"

Have you tried it before, I can't seem to make it work.
